glib-sys/src/lib.rs

```diff
@@ -1399,33 +1554,19 @@ pub struct GIOChannel {
     pub ref_count: c_int,
     pub funcs: *mut GIOFuncs,
     pub encoding: *mut c_char,
-    pub read_cd: GIConv,
-    pub write_cd: GIConv,
-    pub line_term: *mut c_char,
-    pub line_term_len: c_uint,
-    pub buf_size: size_t,
-    pub read_buf: *mut GString,
-    pub encoded_read_buf: *mut GString,
-    pub write_buf: *mut GString,
-    pub partial_write_buf: [c_char; 6],
     _truncated_record_marker: c_void,
-    //use_buffer: guint: 1,
-    //do_encode: guint: 1,
-    //close_on_unref: guint: 1,
-    //is_readable: guint: 1,
-    //is_writeable: guint: 1,
-    //is_seekable: guint: 1,
-    //reserved1: gpointer,
-    //reserved2: gpointer,
+    // /*Ignored*/field read_cd has incomplete type
```

Comment:
Previously that field was considered complete and generated (but other ones later not)

Reply:
This because now `void` types skipped.
```
#[repr(C)]
pub struct GIConv(c_void);

impl ::std::fmt::Debug for GIConv {
    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
        f.debug_struct(&format!("GIConv @ {:?}", self as *const _))
         .finish()
    }
}
```

Reply:
`GIConv` should be a `*mut c_void`:
```c
typedef struct _GIConv *GIConv;
```

Reply:
It was `pub struct GIConv;` here from beginning and changed to `pub struct GIConv(c_void);`  2 year ago in https://github.com/gtk-rs/sys/commit/3a2242b32f8935d6afebac9f0028ce0e55a3adcd.
IMHO this because it had `disguised="1"` and contains no fields

Reply:
Not that I care about this specific one, but it should really be `struct GIConv(*mut c_void)` :) We don't know the fields, but the `GIConv` type is always a pointer. Unlike other cases like `typedef struct _Foo Foo` without `struct _Foo` in any headers

Reply:
Unfortunately disguised="1" is not exactly to be trusted right now: https://bugzilla.gnome.org/show_bug.cgi?id=721481

Reply:
@tmiasko Thanks for info, I even don't know that `disguised="1"` planned only for `typedef struct _X *X;`

Reply:
Added issue https://github.com/gtk-rs/gir/issues/529 as reminder